ubuntu/get-docker.sh

```diff
@@ -70,7 +62,7 @@ then
   echo "-> /etc/apt/keyrings exists"
 else
   echo "-> creating /etc/apt/keyrings"
-  sudo install -m 0755 -d /etc/apt/keyrings
```

Comment:
This was probably fine, just using install to make a directory. Does the mkdir have a flag `-m`?

Reply:
yes, mkdir has an -m flag.